### PR TITLE
1.9/Add-tests-1500-hotfix-permissions-issue-when-downloading-pdf-forms

### PR DESCRIPTION
**Pull request by AlinChindea**
Add tests for the functionality listed in this card - [https://trello.com/c/PKwBrsfX/1500-hotfix-permissions-issue-when-downloading-pdf-forms-e-vvv](https://trello.com/c/PKwBrsfX/1500-hotfix-permissions-issue-when-downloading-pdf-forms-e-vvv)

### CODE REVIEW

File: tests/Unit/Routes/StoreRoutesTest.php [hunk 1]
```diff
@@ -386,7 +388,7 @@ public function testCentreRegistrationsSummaryGate()
             "centre_id" => $this->centre->id,
         ]);
 
-        $route = $specificCentreRoute = URL::route('store.centre.registrations.summary', ['centre' => $this->centre->id]);
```
Comment (charlesstrange2) [inline tests/Unit/Routes/StoreRoutesTest.php]: cant recall why i had it set that way :thinking:

Comment (AlinChindea) [reply tests/Unit/Routes/StoreRoutesTest.php]: don't know 🤔 but removing it doesn't break anything. there's also an unused declaration `use Illuminate\Support\Collection;` that we can remove from this file, I think.

Comment (charlesstrange2) [reply tests/Unit/Routes/StoreRoutesTest.php]: i'm using that to specifically highlight $registrations as collection; on like 21.
It helps with autocomplete in IDE's
Can be removed.

File: tests/Unit/Routes/StoreRoutesTest.php [hunk 2]
```diff
@@ -527,11 +528,20 @@ public function testMVLRouteGuard()
             ->get($route)
             ->assertResponseStatus(403)
         ;
+
+        Auth::logout();
+
+        // Throw a 403 for auth'd but forbidden
+        $this->actingAs($this->downloaderUser, 'store')
```
Comment (charlesstrange2) [inline tests/Unit/Routes/StoreRoutesTest.php]: aha! yes, good spot

**Comment by AlinChindea** [comment]
> Optionally remove references to the Collection class, as you suggested
> otherwise 👍

Removing that unused declarations somehow breaks a test - [see here](https://travis-ci.org/neontribe/ARCVService/builds/651594183?utm_source=github_status&utm_medium=notification) - so I added it back. Merging.

**Pull request merged**